### DISCUSSION TITLE
Scheduling two weekly CG calls

### DISCUSSION THREAD

**Discussion by elf-pavlik** [solid/specification#551] [Ideas]
This discussion is in response to a topic recurring in CG meetings for the last couple of weeks.

> ### Special Meetings

We have a growing list of issues/topics which we would like to process during dedicated meetings. I find it very inefficient to do separate scheduling for each of those topics (I [attempted one already](https://doodle.com/meeting/participate/id/dyX50YVb/vote)).

It might be much more efficient to have two weekly meetings:

* 1 hour long - General weekly CG Meeting, which [is already scheduled](https://github.com/solid/specification/#participation)
* 2 hours long -  Dedicated weekly CG Meeting, each one would be for the topics agreed upon during the general meeting in the previous week

This would allow everyone to have already their calendars set. Based on a specific topic for the next week everyone would simply decide to participate or not, and preferably RSVP in the agenda/scratchpad for that next meeting.

I believe that in those two meetings, we wouldn't need to create task forces most of the time. I have some thoughts about task forces and panels captured in:

* https://github.com/solid/process/issues/324

If possible let's keep this discussion focused on the idea of those two weekly meetings.

**Comment by TallTed** [comment]
Special Topic Calls are used in some of my other WGs, to good effect. I'm in favor of doing so here.

A doodle that starts with the basics of earliest start (7am ET? 6am? 8am? other?) and latest end (10pm ET? 9pm? 11pm? other?) times, and presumes M-F (i.e., no weekend timeslots), is probably the best first step. I suggest using a two-week window for info gathering, as a number of calls are currently running every-other-week, so what looks good one week may be terrible the following week. It will probably be best to use one-hour blocks for info gathering.

A good second step would be for the doodle creator to block out all timeslots already filled by Solid calls (of which there are a few).

Then other participants can tick off their good/if-need-be/bad slots, and we will see what we see...

**Reply by elf-pavlik** [reply]
What you are describing sounds pretty close to what I was trying to do for the first call https://doodle.com/meeting/participate/id/dyX50YVb/vote It only skips Wednesday where we already have a CG call.
Given participation from NA, EU, and Asia I think we actually only have a 4-hour window on any given day to choose the 2 hours from.

**Comment by woutermont** [comment]
Very much in favor of a fixed time slot for dedicated meetings on one-off topics. Would still keep the option open to have task forces concentrate on topics over a longer period of time though.

**Comment by csarven** [comment]
Agree fixed time slot would be practical without a fixed topic area - I mean, let's not redo panel-like meetings! Perhaps with some emphasis on being practical, co-hacking, task force like topics focusing on something specific. (Still in recent memory, Pavlik, Rahul, and I had great several sessions while working on a release of the Notifications Protocol).

Note that the current two proposed special topic calls (in the CG weekly) are both on Tuesday... (but it can be another day that doesn't conflict with main CG weekly and possibly WG but we'll see that later). I'd normally suggest to go with 14:00 UTC to start things off, but participation in these kinds of meetings vary over time, and we should accommodate times that are more suitable to folks on the opposite edges of the map :world_map: https://www.timeanddate.com/time/map/ :)

:information_source:  [Guidelines for the Organization of Fully Online Meetings](https://www.ietf.org/rfc/rfc9400.html)

**Reply by elf-pavlik** [reply]
For 2 hour meeting, I think we can't go outside of starting it between 12:00 UTC and 15:00 UTC which seems at most 4 options each day. When we eliminate weekends and Wednesdays, we have top 4 x 4 = 16 options to choose from. At least this was my reasoning behind: https://doodle.com/meeting/participate/id/dyX50YVb/vote

**Comment by csarven** [comment]
Created https://github.com/solid/specification/discussions/555 and added a recurring event in the CG calendar . Can this discussion be closed?

**Reply by woutermont** [reply]
I guess so ... @elf-pavlik ?
